prov/efa/src/efa_base_ep.h

```diff
@@ -64,6 +64,44 @@
 #include "ofi_util.h"
 #include "ofi_file.h"
 
+extern struct fi_provider efa_prov;
+
+#define EFA_PROV_NAME "efa"
```

Comment:
These macros and `efa_prov` should stay in `efa.h`. Why do you need to move it?

Reply:
I only moved things necessary for `efa_base_ep` now